src/components/experiment/experiment.js

```diff
@@ -101,23 +117,40 @@ export default class Experiment extends React.Component {
     // These will trigger `componentWillReceiveProps`
     dispatchActivate({experiment});
     dispatchPlay({experiment, variation});
-    this.setState({ experiment, variation });
+    this.setState({ experiment, variation, played: true });
+  }
+
+  isNotVariationChildren() {
+    const { children } = this.props;
+    return (React.Children.count(children) === 1 && (!React.isValidElement(children) || React.Children.only(children).type !== VariationComponent || React.Children.only(children).type !== VariationContainer))
```

Comment:
`&& ![VariationComponent, VariationContainer].includes?(React.Children.only(children).type` as the last clause in this conditional (after or)? This way we call `only` once